.pipelines/shared-cluster.yml

```diff
@@ -0,0 +1,95 @@
+# No PR triggers as it runs on a schedule
+pr: none
+trigger: none
+
+schedules:
+- cron: "0 1 * * Sun"
```

Comment:
As we already seen how much we need shared cluster "to stay" when we see bugs, should we remove this and make it manually triggered pipeline?